### ISSUE TITLE
`main.nf` missing

### ISSUE THREAD

**Issue by ggabernet** [wassermanlab/Variant_catalogue_pipeline#18]
Hi all,

this is an interesting pipeline, thanks for developing it!

I just came across a couple of points when I wanted to test it that I think would make it easier for users to use the pipeline.

Naming the main nextflow pipeline file exactly `main.nf`, would make it possible to run the pipeline directly from the GitHub repository name like so:

```
nextflow run wassermanlab/Variant_catalogue_pipeline
```

Currently one needs to clone the pipeline before running it.

What would be the command needed to test the pipeline?

**Comment by scorreard** [comment]
Hi Gisela,

Sorry for the late answer. We renamed the main nextflow pipeline, thanks for the suggestion.

For testing, I would recommend trying a more portable version I worked on (the code is exactly the same, and it includes testdata from the nf-core/raredisease team which works well for the variantcatalogue pipeline) : https://github.com/scorreard/Variant_catalogue_pipeline.
 It will break at the Hail step as there is no container for this (This is what I am trying to figure out now) (https://hail.is/)

Hopefully, I will merge both repo soon for more clarity.

I hope this helps,
Solenne
